modules/fluctBidAdapter.js

```diff
@@ -0,0 +1,121 @@
+import * as utils from '../src/utils';
+import { registerBidder } from '../src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'fluct';
+const END_POINT = 'https://hb.adingo.jp/prebid';
+const VERSION = '1.2';
+const NET_REVENUE = true;
+const TTL = 300;
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['adingo'],
+
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {BidRequest} bid The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: (bid) => {
+    return !!(bid.params.groupId && bid.params.tagId && bid.params.dfpUnitCode);
```

Comment:
(talked directly)

lets make `dfpUnitCode` as optional :)

`dfpUnitCode` is as is optional current hb.adingo.jp internaly and/also prebid adapter should work admanager other than DFP(gpt).